scripts/api.js

```diff
@@ -22,7 +22,10 @@ function handleRequestWithRetry(requestFn, options, callbackData, callbacks) {
     } catch (error) {
         if (error.additionalInfo.status === 401) {
             sys.logs.info("[zoho] Handling request...: "+ JSON.stringify(error));
-            dependencies.oauth.functions.refreshToken('zoho:refreshToken');
+            const accessTokenResponse = httpService.post({url: config.get("ZOHO_OAUTH_API_BASE_URL")+"/oauth/v2/token?refresh_token=" + sys.storage.get('installationInfo-Zoho-User-'+sys.context.getCurrentUserRecord().id() + ' - refresh_token', {decrypt:true}) + "&client_id=" + config.get().clientId + "&client_secret=" + config.get().clientSecret + "&redirect_uri=" + config.get().oauthCallback + "&grant_type=refresh_token"});
```

Comment:
Let's split a little the code and use the same format to get config values
```suggestion
            const actualRefreshToken = sys.storage.get('installationInfo-Zoho-User-'+sys.context.getCurrentUserRecord().id() + ' - refresh_token', {decrypt:true});
            const url = config.get("ZOHO_OAUTH_API_BASE_URL")+"/oauth/v2/token?refresh_token=" + actualRefreshToken + "&client_id=" + config.get("clientId") + "&client_secret=" + config.get("clientSecret") + "&redirect_uri=" + config.get("oauthCallback") + "&grant_type=refresh_token";
            const accessTokenResponse = httpService.post({url: url});
```

Reply:
@jsanchez-slingr FYI I'm commiting this and creating a new tag after merge